simulariumio/cytosim/cytosim_converter.py

```diff
@@ -191,7 +191,6 @@ def _parse_objects(
         used_unique_IDs: List[int],
         overall_line: int,
         total_lines: int,
-        scale_factor: float = None,
```

Comment:
Just curious: what's deciding the scale_factor should be a float or not

Reply:
So scale_factor is an optional input parameter that the user may or may not provide when setting up their `CytosimData` object.  Then, in `TrajectoryConverter.scale_agent_data()`, which I didn't change in this PR, if scale_factor is None, we generate a scale factor to get the data to look decent in the viewer (so like not be so tiny you can't see anything or so huge that it's out of frame).